tool/lib/commands/pub_get.dart

```diff
@@ -62,26 +63,19 @@ class PubGetCommand extends Command {
 
       final progress = log.progress('  ${p.relativePath}');
 
-      final process = await Process.start(
-        sdk.flutterToolPath,
-        ['pub', command],
+      final process = await processManager.runProcess(
+        CliCommand.flutter(
+          'pub $command',
+        ),
         workingDirectory: p.packagePath,
       );
-      final stderr = process.stderr;
-
-      final exitCode = await process.exitCode;
 
+      final exitCode = process.exitCode;
       if (exitCode == 0) {
         progress.finish(showTiming: true);
       } else {
```

Comment:
I don't think we will ever hit this else case because the above command will have `throwOnException = true`

Reply:
Ah yes - should I remove this code or pass `throwOnException : false`? The current behaviour is to just print and return the failure count from `run` (which presumably becomes the exit code), whereas with `throwOnException: true` it'll presumably always get the same exit code (and not get the same printing we had).

Reply:
How about we change `DevToolsProcessManagerExtension.runProcess` to include the std err output on exceptions? then we just get this stderr handling for free on all the process runs

Reply:
I realise there's another difference if we do that - it'll stop on the first failure. The current behaviour is to continue to run the others so you can see all the results up front (this would avoid getting a single error, fixing it, then finding the next package also has a failure).

So I think `throwOnException:false` might make most sense for these loops?